content/source/docs/enterprise/private/automating-the-installer.html.md

```diff
@@ -292,6 +292,13 @@ while ! curl -ksfS --connect-timeout 5 https://tfe.example.com/_health_check; do
 done
 ```
 
+## If the installation does not appear to be configured correctly
+
+If the installation script (`install.sh`) exits successfully, but the Replicated web UI prompts for additional configuration rather than skipping that step, it's likely that the supplied configuration files were not applied during installation.
+
+* Verify the locations and permissions of the files: `/etc/replicated.conf` for the Replicated configuration file, and the path specified in the Replicated configuration file for the application settings. 
+* Run a JSON validator on the files to check their validity.
```

Comment:
After fixing what was wrong with the files, do you run the install script again? Or just go to the web UI?

Reply:
Ah yeah, that's a good call as something to add. To verify that they work, you'd need to re-run it. I'll update.